node/modules/rpc.go

```diff
@@ -0,0 +1,66 @@
+package modules
+
+import (
+	"context"
+
+	rpc "github.com/libp2p/go-libp2p-gorpc"
+	"github.com/libp2p/go-libp2p/core/host"
+	"github.com/libp2p/go-libp2p/core/peer"
+	"github.com/libp2p/go-libp2p/core/protocol"
+
+	"github.com/filecoin-project/lotus/api"
+	"github.com/filecoin-project/lotus/chain/types"
+	consensus "github.com/filecoin-project/lotus/lib/consensus/raft"
+	"github.com/filecoin-project/lotus/node/impl/full"
+)
+
+type RPCHandler struct {
+	mpoolAPI full.MpoolAPI
+	cons     *consensus.Consensus
+}
+
+//type ConsensusRPCAPI struct {
+//	cons       *consensus.Consensus
+//	rpcHandler *RPCHandler
+//}
```

Comment:
```suggestion
```

Reply:
Will remove all the commented out stuff once we get to a good final state